src/builtins.ts

```diff
@@ -2997,7 +2997,7 @@ function builtin_function_call(ctx: BuiltinContext): ExpressionRef {
     compiler.currentType = returnType;
     return compiler.module.unreachable();
   }
-  var indexArg = compiler.compileExpression(assert(ctx.thisOperand), ftype, Constraints.CONV_IMPLICIT);
+  var tableIndexExpr = compiler.compileExpression(assert(ctx.thisOperand), ftype, Constraints.CONV_IMPLICIT);
```

Comment:
Even though a builtin as well, this one is the implementation of `Function#call`, essentially doing an idiomatic call by function pointer. Only `builtin_call_indirect` is special.